# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1296552868 18000
#      Tue Feb 01 04:34:28 2011 -0500
# Node ID d6417ee864c9cf1f543455e5082efa5b0669fa81
# Parent  e7c8ce0c0d3e2af603d1cc59bfa9fb2d3822047b
scatter.m: pass h to set, not ax

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2011-02-01  John W. Eaton  <jwe@octave.org>
+
+	* plot/scatter3.m: Pass h to set, not ax.
+
 2011-01-31  John W. Eaton  <jwe@octave.org>
 
 	* plot/private/__axis_label__.m: Don't call __fltk_redraw__.
 
 2011-01-31  John W. Eaton  <jwe@octave.org>
 
 	* plot/__fltk_ginput__.m: Remove line that should have been
 	removed in previous change.
diff --git a/scripts/plot/scatter3.m b/scripts/plot/scatter3.m
--- a/scripts/plot/scatter3.m
+++ b/scripts/plot/scatter3.m
@@ -66,17 +66,17 @@ function retval = scatter3 (varargin)
       newplot ();
       tmp = __scatter__ (h, 3, "scatter3", varargin{:});
     unwind_protect_cleanup
       axes (oldh);
     end_unwind_protect
   endif
 
   if (! ishold ())
-    set (ax, "view", [-37.5, 30],
+    set (h, "view", [-37.5, 30],
          "xgrid", "on", "ygrid", "on", "zgrid", "on");
   endif
 
   if (nargout > 0)
     retval = tmp;
   endif
 
 endfunction
